doc/api/next_api_changes/2019-03-27-AL.rst

```diff
@@ -0,0 +1,8 @@
+Deprecations
+````````````
+
+The `\mathcircled` mathtext command (which is not a real TeX command)
+is deprecated.  Directly use unicode characters (e.g.
+``"\N{SMALL LATIN CAPITAL LETTER A}"`` or ``"\u24b6"``) instead.
```

Comment:
Isn't this `CIRCLED LATIN CAPITAL LETTER A`?

Reply:
oops, fixed